tests/Directory.Build.targets

```diff
@@ -19,12 +21,42 @@
 
   <Target Name="ZipTestArchive" AfterTargets="Build" Condition="'$(ArchiveTests)' == 'true' and '$(RunTestsOnHelix)' == 'true'">
     <Error Condition="'$(TestArchiveTestsDir)' == ''" Text="TestArchiveTestsDir property to archive the test folder must be set." />
+    <PropertyGroup>
+      <TestsArchiveSourceDir Condition="'$(TestsArchiveSourceDir)' == ''">$(OutDir)</TestsArchiveSourceDir>
+    </PropertyGroup>
 
     <MakeDir Directories="$(TestArchiveTestsDir)" />
-    <ZipDirectory SourceDirectory="$(OutDir)"
+    <ZipDirectory SourceDirectory="$(TestsArchiveSourceDir)"
                   DestinationFile="$([MSBuild]::NormalizePath('$(TestArchiveTestsDir)', '$(MSBuildProjectName).zip'))"
                   Overwrite="true" />
   </Target>
 
-  <Import Project="Shared\Aspire.Workload.Testing.targets" Condition="'$(IsWorkloadTestProject)' == 'true'" />
+  <!-- Used for running one helix job per test class -->
+  <Target Name="_ExtractTestClassNames"
```

Comment:
Why do we want one helix job per test class? That seems extreme, isn't it?

Reply:
This is used for workload tests, where the tests are expensive. I'm just making it available for other projects.